server/utils/projects.js

```diff
@@ -0,0 +1,29 @@
+// Projects Validations
+const { isValidStr, isValidObj, isValidArray } = require('./index');
+const isValidProjectName = (projectName) => {
+	const projectName = isValidStr(projectName, 'project name', 'min', 3);
+	return projectName;
+};
+const isValidProjectObject = (projectObject) => {
+	isValidObj(projectObject);
+	return {
+		name: isValidProjectName(projectObject.name, 'Project name', 'min', 3),
+		description: projectName.description,
+		github: projectName.github,
```

Comment:
for now, add valid string validation
again this could be `null` too. it is an optional field